pkg/plugins/golang/v3/scaffolds/internal/templates/makefile.go

```diff
@@ -62,6 +62,8 @@ const makefileTemplate = `
 IMG ?= {{ .Image }}
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
 CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false"
+# K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
+K8S_VERSION = 1.21
```

Comment:
I'd rather be explicit about this variable name, since the project itself should work with a range of k8s versions regardless of what binaries envtest is using.
```suggestion
# ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
ENVTEST_K8S_VERSION = 1.21
```